### PR TITLE
Update links in 2 files

### PR DESCRIPTION
**Pull request by vj-codes**
#### Is your pull request related to a problem? Please describe
Fixed issue #92

#### Checklist
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **[CONTRIBUTING](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/CONTRIBUTING.md)** document.
- [ ] All new and existing tests passed.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -41,5 +41,4 @@ outlined on that page and do not file a public issue.
 
 ## License
 By contributing to Open Source Education Bot, you agree that
-your contributions will be licensed under the LICENSE file in the root directory
-of this source tree.
\ No newline at end of file
+your contributions will be licensed under the [LICENSE file](https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/LICENSE).
```
Comment (aboullaite) [inline CONTRIBUTING.md:44]: you can simply say under MIT licence

Comment (vj-codes) [reply CONTRIBUTING.md:44]: and keep the link as it is right?

Comment (hoduchieu01) [reply CONTRIBUTING.md:44]: @vj-codes you should use the link as **(./LICENSE)**

File: README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ The main purpose of this repository is to continue evolving open source. We want
 Facebook has adopted a Code of Conduct that we expect project participants to adhere to.
 Please read the [full text][code] so that you can understand what actions will and will not be tolerated.
 
-[code]: https://code.fb.com/codeofconduct/
+[code]: https://github.com/fbdevelopercircles/open-source-edu-bot/blob/master/CODE_OF_CONDUCT.md
```
Comment (hoduchieu01) [inline README.md:136]: @vj-codes  you should use the link as a simple version **(./CODE_OF_CONDUCT.md)**

**Comment by vj-codes** [comment]
@hoduchieu01 @aboullaite done!

**Pull request merged**